Content.Client/Content.Client.csproj

```diff
@@ -23,6 +23,9 @@
     <ProjectReference Include="..\RobustToolbox\Robust.Client\Robust.Client.csproj" />
     <ProjectReference Include="..\Content.Shared\Content.Shared.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="GameObjects\Components\Utensil" />
+  </ItemGroup>
```

Comment:
Why did you add this? It's completely useless. The project automatically recognizes your cs files even in new folders.

Reply:
Good question, must have been by accident.